tests/e2e/e2e_ibc_test.go

```diff
@@ -64,103 +92,87 @@ func (s *E2ETest) TestIBCTokenTransfer() {
 
-		// send uatom from gaia to umee
-		// Note : gaia -> umee (ibc_quota will not check token limit)
+		// compute the amount of ATOM sent from umee to gaia which would meet atom's token quota
 		atomPrice, err := s.QueryHistAvgPrice(umeeAPIEndpoint, atomSymbol)
 		s.Require().NoError(err)
-		emOfAtom := sdk.NewDecFromInt(totalQuota).Quo(atomPrice)
-		c := sdk.NewInt64Coin("uatom", emOfAtom.Mul(powerReduction).RoundInt64())
-		s.Require().True(atomPrice.GT(sdk.OneDec()), "price should be non zero, and expecting higher than 1, got: %s", atomPrice)
-		s.Require().True(c.Amount.GT(sdk.NewInt(2_000_000)), "amount should be non zero, and expecting much higher than 2 atom = 2e6 uatom, got: %s", c.Amount)
+		s.Require().True(atomPrice.GT(sdk.OneDec()),
+			"atom price should be non zero, and expecting higher than 1, got: %s", atomPrice)
+		atomQuota := sdk.NewCoin(uatomIBCHash,
+			sdk.NewDecFromInt(tokenQuota).Quo(atomPrice).Mul(powerReduction).RoundInt(),
+		)
 
-		s.SendIBC(setup.GaiaChainID, s.Chain.ID, "", c, false)
-		s.checkSupply(umeeAPIEndpoint, uatomIBCHash, c.Amount)
+		// send $500 ATOM from gaia to umee. (ibc_quota will not check token limit)
+		atomFromGaia := mulCoin(atomQuota, "5.0")
+		atomFromGaia.Denom = "uatom"
+		s.SendIBC(setup.GaiaChainID, s.Chain.ID, "", atomFromGaia, false, "")
+		s.checkSupply(umeeAPIEndpoint, uatomIBCHash, atomFromGaia.Amount)
 
-		// sending more tokens than token_quota limit of umee (token_quota is 100$)
-		histoAvgPriceOfUmee, err := s.QueryHistAvgPrice(umeeAPIEndpoint, umeeSymbol)
+		// compute the amout of UMEE sent to gaia which would meet umee's token quota
+		umeePrice, err := s.QueryHistAvgPrice(umeeAPIEndpoint, umeeSymbol)
 		s.Require().NoError(err)
-		exceedAmountOfUmee := sdk.NewDecFromInt(totalQuota).Quo(histoAvgPriceOfUmee)
-		s.T().Logf("sending %s amount %s more than %s", umeeSymbol, exceedAmountOfUmee.String(), totalQuota.String())
-		exceedAmountCoin := sdk.NewInt64Coin(appparams.BondDenom, exceedAmountOfUmee.Mul(powerReduction).RoundInt64())
-		s.SendIBC(s.Chain.ID, setup.GaiaChainID, "", exceedAmountCoin, true)
-		// check the ibc (umee) quota after ibc txs
+		s.Require().True(umeePrice.GT(sdk.MustNewDecFromStr("0.001")),
+			"umee price should be non zero, and expecting higher than 0.001, got: %s", umeePrice)
+		umeeQuota := sdk.NewCoin(appparams.BondDenom,
+			sdk.NewDecFromInt(tokenQuota).Quo(atomPrice).Mul(powerReduction).RoundInt(),
+		)
+
+		// send $90 UMEE from umee to gaia (ibc_quota will check)
+		// Note: receiver is null so hermes will default send to key_name (from config) of target chain (gaia)
+		sendUmee := mulCoin(umeeQuota, "0.9")
+		s.SendIBC(s.Chain.ID, setup.GaiaChainID, "", sendUmee, false, fmt.Sprintf(
+			"sending %s (less than token quota) ", sendUmee.String()))
+		s.checkOutflows(umeeAPIEndpoint, appparams.BondDenom, true, sdk.NewDecFromInt(sendUmee.Amount), appparams.Name)
+		s.checkSupply(gaiaAPIEndpoint, umeeIBCHash, sendUmee.Amount)
```

Comment:
@robert-zaremba Can you think of any reason this would be failing?

Currently `checkSupply` is showing zero here, probably meaning that the IBC transfer failed or was blocked by quota